htdocs/core/lib/functions.lib.php

```diff
@@ -15887,6 +15893,10 @@ function show_actions_messaging($conf, $langs, $db, $filterobj, $objcon = null,
 			$sql .= ", " . MAIN_DB_PREFIX . "expedition as o";
 		} elseif (is_object($filterobj) && get_class($filterobj) == 'Propal') {
 			$sql .= ", " . MAIN_DB_PREFIX . "propal as o";
+		} else {
+			if (is_object($filterobj) && !empty($filterobj->table_element) && !empty($filterobj->element) && !empty($filterobj->id)) {
```

Comment:
Missing test
&& array_key_exists('ref', $filterobj->fields) to have same condition than on line 15850